modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -24,16 +26,48 @@ export default function DropdownMenu({
 	menuProps,
 	portletId: _portletId,
 	portletNamespace: _portletNamespace,
+	searchable,
```

Comment:
It's interesting to consider if it's worth adding new props as tag attributes, along with getters and setters. As it currently is in PR, we are using dynamic attributes mechanism. I think this is fine as it is, because:
1. All three attributes are directly passed to the underlying `ClayDropDownWithItems`
2. None of the three attributes is relevant in SSR.

Reply:
I agree with using dynamic props in this case, since as you comment, they are not relevant props for the SSR.